.devcontainer/devcontainer.json

```diff
@@ -2,41 +2,41 @@
 // https://github.com/microsoft/vscode-dev-containers/blob/master/containers/docker-from-docker/.devcontainer/devcontainer.json
 // For format details, see https://aka.ms/vscode-remote/devcontainer.json
 {
-    "name": "Fluid",
-    "build": {
-        "dockerfile": "Dockerfile",
-        // Update 'NODE_VERSION' to pick a Node version: 10, 12, 14
-        "args": {
-            "NODE_VERSION": "14"
-        }
-    },
-    "runArgs": [ "--init" ],
-    "mounts": [ "source=/var/run/docker.sock,target=/var/run/docker-host.sock,type=bind" ],
-    "overrideCommand": false,
+	"name": "Fluid",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'NODE_VERSION' to pick a Node version: 10, 12, 14
+		"args": {
+			"NODE_VERSION": "14"
```

Comment:
Probably need to update to 16?

Reply:
I don't use the devcontainer so I don't know how to test it, but yes, I suspect that would be a good change. I'll make it.

Reply:
Lower down in the file we install node using nvm to get the preferred version. I don't get why we use both and as I'm not setup to test this, I'm going to leave it as is.